server/game/player.js

```diff
@@ -213,6 +226,20 @@ class Player extends Spectator {
         this.game.raiseEvent('onUsedPlotsModified', { player: this });
     }
 
+    modifyUsedPlotsWithTrait(value, trait) {
+        if(value >= 0) {
+            for(let i = 0; i < value; i++) {
+                this.usedPlotsModifierByTrait.add(trait);
+            }
+        } else {
+            for(let i = 0; i < value * -1; i++) {
+                this.usedPlotsModifierByTrait.remove(trait);
+            }
+        }
+        
+        this.game.raiseEvent('onUsedPlotsModified', { player: this });
```

Comment:
This event isn't used anywhere. Were you finding that effects weren't being recalculated when adding / removing Gulltown from play? If so, I'd recommend adding a TODO comment here to fix those calculations. If not, we should get rid of it.

Reply:
Might be we don´t need that event at all, i just copied it from the already present modifyUsedPlots method above without looking for references to the event. I will double check if it is needed to update the game state for cards referencing the used plot counter and post my findings

Reply:
Tested the interaction between Host of the Boneway, Ricasso and Gulltown without this line of code (and the other one in the method above) and it didn´t change the behaviour. The Host´s STR was increased as expected.
=> i removed the two lines